File: Java/lab11/src/main/java/org/example/Order.java
package org.example;

import java.io.FileWriter;
import java.io.IOException;

public class Order {
    private final String nameOrder;
    private final String typeOfOrder;
    public Order(String nameOrder, String typeOfOrder) {
        this.nameOrder = nameOrder;
        this.typeOfOrder = typeOfOrder;
    }
    public  void createOrder(){
        try (FileWriter writer = new FileWriter("orders.txt", true)) {
            writer.write(nameOrder+","+typeOfOrder+"\n");
        } catch (IOException e) {
            e.printStackTrace();
        }
    }
}

File: Python/lab11/Order.py
class Order:
    def __init__(self, name_order, type_of_order):
        self.name_order = name_order
        self.type_of_order = type_of_order

    def create_order(self):
        with open('orders.txt', 'a') as file:
            file.write(f'{self.name_order},{self.type_of_order}\n')